utility/retrieve_polarion_ids.py

```diff
@@ -0,0 +1,141 @@
+usage = """Retrieve Polarion Ids from test suites.
```

Comment:
Could we reword the statement? I believe we are determining if the test cases implemented are inline with the stated guidelines of having 1 to 1 mapping with a Test case.

Reply:
Done